specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/alerts.json

```diff
@@ -0,0 +1,1022 @@
+      "additionalProperties": true,
+      "description": "Links related to the alert"
+    },
+    "AlertExtendedProperties": {
+      "type": "object",
+      "additionalProperties": true,
+      "description": "Custom properties for the alert."
+    },
+    "AlertList": {
+      "type": "object",
+      "description": "List of security alerts",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "describes security alert properties.",
+          "items": {
+            "$ref": "#/definitions/Alert"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page."
+        }
+      }
+    },
+    "AlertProperties": {
+      "type": "object",
+      "description": "describes security alert properties.",
+      "properties": {
+        "alertType": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Unique identifier for the detection logic (all alert instances from the same detection logic will have the same alertType)."
+        },
+        "systemAlertId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Unique identifier for the alert."
+        },
+        "productComponentName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of Azure Security Center pricing tier which powering this alert. Learn more: https://docs.microsoft.com/en-us/azure/security-center/security-center-pricing"
+        },
+        "alertDisplayName": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The display name of the alert."
+        },
+        "description": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Description of the suspicious activity that was detected."
+        },
+        "severity": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "Informational",
+            "Low",
+            "Medium",
+            "High"
+          ],
+          "x-ms-enum": {
+            "name": "alertSeverity",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Informational",
+                "description": "Informational"
+              },
+              {
+                "value": "Low",
+                "description": "Low"
+              },
+              {
+                "value": "Medium",
+                "description": "Medium"
+              },
+              {
+                "value": "High",
+                "description": "High"
+              }
+            ]
+          },
+          "description": "The risk level of the threat that was detected. Possible Values: Informational, Low, Medium, and High. Learn more: https://docs.microsoft.com/en-us/azure/security-center/security-center-alerts-overview#how-are-alerts-classified."
```

Comment:
[nit] "Possible Values: Informational, Low, Medium, and High." is unnecessary here in the description as you're modeling the possible values in the enum.

Reply:
Removed from comment